rskj-core/src/test/java/co/rsk/peg/BridgeStorageProviderTest.java

```diff
@@ -83,7 +83,8 @@
 public class BridgeStorageProviderTest {
     private static final byte FAST_BRIDGE_FEDERATION_SCRIPT_HASH_TRUE_VALUE_TEST = (byte) 1;
     private static final DataWord NEW_FEDERATION_BTC_UTXOS_KEY = DataWord.fromString("newFederationBtcUTXOs");
-    private static final DataWord NEW_FEDERATION_BTC_UTXOS_KEY_FOR_TESTNET = DataWord.fromString("newFederationBtcUTXOsForTestnet");
+    private static final DataWord NEW_FEDERATION_BTC_UTXOS_KEY_FOR_TESTNET_PRE_HOP = DataWord.fromString("newFederationBtcUTXOsForTestnet");
```

Comment:
Why not add the suffix "PreHop" to this key instead of adding "PostHop" to the one below?

Reply:
Changing the existing key would fork the network, since UTXOs would get saved in storage under a different key. If treated as a consensus change (one key pre Hop, the other one post Hop) then when Hop activates we would lose all the registered UTXOs since there would be none under the new key.